WORKSPACE

```diff
@@ -103,24 +103,11 @@ gcs_file(
 
 load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
 
-rbe_autoconfig(name = "rbe_default")
```

Comment:
Update //bazelrc/.bazelrc to ensure it keeps working with this change? I intend to use it in my GCB job to test rbe_autoconfig

Reply:
re-added the target.